invenio_rdm_records/contrib/codemeta/__init__.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+"""Custom fields implementation for CodeMeta."""
```

Comment:
```suggestion
"""CodeMeta contrib module."""
```

nit: since there might be more things added in the future.